tests/Jellyfin.Server.Implementations.Tests/Test Data/Updates/manifest-stable.json

```diff
@@ -13,15 +13,17 @@
                 "targetAbi": "10.6.0.0",
                 "sourceUrl": "https://repo.jellyfin.org/releases/plugin/anime/anime_10.0.0.0.zip",
                 "checksum": "93e969adeba1050423fc8817ed3c36f8",
-                "timestamp": "2020-08-17T01:41:13Z"
+                "timestamp": "2020-08-17T01:41:13Z",
+                "assemblies": [ "Jellyfin.Plugin.Anime.dll" ]
```

Comment:
Why would the assemblies information need to be present in the repo manifest?
Would it not be more appropriate to only include it in the plugin metadata (`meta.json`)?

The content of the items in the versions array and the `meta.json` file in the plugin archive are not exact duplicates.

```json
{
    "category": "Metadata",
    "changelog": "### Bug Fixes ###\n- fix client string (#19) @Shadowghost\n\n### Code or Repo Maintenance ###\n- Fix readme (#20) @Shadowghost",
    "description": "This plugin provides images from the Cover Art Archive https://musicbrainz.org/doc/Cover_Art_Archive and depends on the MusicBrainz metadata provider to know what images belong where\n",
    "guid": "8119f3c6-cfc2-4d9c-a0ba-028f1d93e526",
    "imageUrl": "https://repo.jellyfin.org/releases/plugin/images/jellyfin-plugin-coverartarchive.png",
    "name": "Cover Art Archive",
    "overview": "MusicBrainz Cover Art Archive",
    "owner": "jellyfin",
    "targetAbi": "10.8.0.0",
    "timestamp": "2022-01-08T17:22:30Z",
    "version": "7.0.0.0"
}
```

Further I'd like to note that the Anime plugin is deprecated, and as such might not be the best test target (it was split into separate plugins, and is no longer maintained)

Reply:
The manifest is the repository that these are drawn from. The `meta.json` is ultimately derived from the version information supplied by the `manifest.json` seen here: 
https://github.com/jellyfin/jellyfin/blob/bb5bf0277ae33fd4668e00bd2f1a85e89aef466a/Emby.Server.Implementations/Plugins/PluginManager.cs#L413-L428

I don't go out of my way to change the current behavior between the `manifest.json` and `meta.json` in the case that the `meta.json` already exists. Since my manual testing involved supplying the `meta.json` directly, it seems like it would work in either case. 

The use of the Anime plugin was merely out of convenience. The `manifest-stable.json` is the test data response used in other testing. I wanted to have a blend of packages _with_ assemblies and those _without_. If this still feels inappropriate, I can re-assess.